package.json

```diff
@@ -18,7 +18,7 @@
     "cy:component": "cypress run --component",
     "cy:component:chrome": "cypress run --component --browser chrome",
     "clean": "git clean -fdx",
-    "ci:publish": "pnpm publish -r --access public"
+    "ci:publish": "pnpm --filter=!vtk-js-transfer-function-example publish -r --access public"
```

Comment:
Does in work to just add `private: true` to vtk-js-transfer-function-example's package.json?

Reply:
Not sure.  Would be nice if packages could opt out.  I'ed guess it would error.  

Maybe we try for a clean publish and expriment after?